dashboards-observability/.cypress/integration/app_analytics.spec.js

```diff
@@ -481,30 +483,30 @@ describe('Viewing application', () => {
   it('Hides application visualizations in Event Analytics', () => {
     cy.visit(`${Cypress.env('opensearchDashboards')}/app/observability-dashboards#/event_analytics`);
     cy.wait(delay*3);
-    if (cy.get('.euiFieldSearch').length > 1) {
+    if (cy.get('.euiButton').length == 2) {
```

Comment:
why would the number of buttons change?

Reply:
If there are no queries or visualizations, the empty event analytics page has two more buttons for creating